content/tokio/tutorial/hello-tokio.md

```diff
@@ -13,8 +13,8 @@ then read back the key. This will be done using the Mini-Redis client library.
 Let's start by generating a new Rust app:
 
 ```bash
-$ cargo new my-redis
-$ cd my-redis
+cargo new my-redis
+cd my-redis
 ```
```

Comment:
I don't think these should be taken out. Starting a line with $ is a convention for saying "this is a bash command". (And it can be used to distinguish between commands and their output.)